File: JAVASCRIPT/CLASES/js/Vendedor.js
class Vendedor extends Trabajador{

    static VENTAS = 120000.0;
    constructor(id, nombre, area, comision){
        super(id, nombre, area);
        this.comision = comision;
    }
    

    getComision(){
        return this._comision;
    }

    salario(){
        return (ventas * (comision/100)) + ventas;
    }
}

File: 20242/Clase 16_09/Herencia/Vendedor.java
public class Vendedor extends Trabajador{

    private double comision;
    public final double ventas = 120000.0;

    public Vendedor(int id, String nombre, String area, double comision){
        super(id, nombre, area);
        this.comision = comision;

    }
    
    public double getComision(){
        return comision;
    }

    public double salario(){
        return (ventas * (comision/100)) + ventas;
    }
}